CMakeLists.txt

```diff
@@ -87,6 +87,8 @@ IF( ${NO_QTMULTIMEDIA} )
    ADD_DEFINITIONS( -DNO_QTMULTIMEDIA )
 ENDIF()
 
+ADD_DEFINITIONS("-std=c++11")
```

Comment:
I think you aren't adding a precompiler definition here, and modifying `CMAKE_CXX_FLAGS` with this option will only work with gcc/clang. I think we want to do [this](https://crascit.com/2015/03/28/enabling-cxx11-in-cmake/)

    set(CMAKE_CXX_STANDARD 11)
    set(CMAKE_CXX_STANDARD_REQUIRED ON)